docs/guides/component-testing/third-party-definitions.mdx

```diff
@@ -0,0 +1,245 @@
+  `,
+});
+```
+
+Our Framework Definition shows up in Cypress!
+
+<DocsImage
+  src="/img/guides/component-testing/framework-definition-1.png"
+  alt="custom framework definition"
+/>
+
+We defined the `dependencies`, which are also correctly handled - we haven't installed them all, so Cypress is prompting us to do so:
+
+
+<DocsImage
+  src="/img/guides/component-testing/framework-definition-2.png"
+  alt="required dependencies"
+/>
+
+## Mount Adapter
+
+The second part of defining a Framework Definition is the Mount Adapter. This is the function that renders the component in your tests using `cy.mount()`. 
+
+By default, Cypress will look for this as a `mount` function that is a **named export** from the package. This should be written in a `index.mjs` file. This example is for a [Solid.js](https://www.solidjs.com/) mount adapter:
+
+```js
+import { getContainerEl, setupHooks } from "@cypress/mount-utils";
+import { render } from "solid-js/web";
+
+let dispose;
+
+function cleanup() {
+  dispose?.();
+}
+
+/**
+ * @param {() => JSX.Element} - component to render
+ */
+export function mount(component, options = {}) {
+  const root = getContainerEl();
+  dispose = render(() => component, root);
+
+  return cy.wait(0, { log: false }).then(() => {
```

Comment:
Might be good to explain what's going on with the `cy.wait` - the async render workaround probably isn't obvious to most people